rstudio/rstudio
# Fixed missing file-type icons on Electron Mac desktop

### Intent

Addresses [File-type icons not showing for RStudio Desktop on macOS #12252](https://github.com/rstudio/rstudio/issues/12252)

### Approach

The icon files were already being registered in Info.plist (exactly the same as with the Qt-version of RStudio), but the actual icon files were not being installed. Now they are.

### Automated Tests

None

### QA Notes

Finder on macOS can be finicky about when it refreshes the file icons supplied by an application, but so long as you install RStudio.app to /Applications, run it once, then logout and back in (and possibly perform some other arcane rituals mentioned in the issue) they should start to show up again.

I did not do anything about giving RStudio priority over other applications that might support some file types; for example, XCode registers support for .R file so you might see its icon instead of ours, etc.

Also, Desktop Pro (Qt) was installing the icons, so ideally test on a machine that doesn't have RDP.

### Documentation

None

### Checklist
